frontend/src/components/program/picasso/PicassoEntry.vue

```diff
@@ -174,6 +179,23 @@ export default {
     isLongDuration() {
       return this.scrollHeight <= this.clientHeight
     },
+    durationText() {
+      const start = this.$date(this.scheduleEntry.startTimestamp).utc(true)
+      const end = this.$date(this.scheduleEntry.endTimestamp).utc(true)
+      if (this.dateShort(start) === this.dateShort(end)) {
```

Comment:
This logic could also go into `dateHelperUTCFormatted`. Likely, we use the logic again somewhere else.